memorygame.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="memorygame.css">
+</head>
+<body>
+  <div class="container">
+  <div class="card" id="cardId">
+  <div class="cover front">
+  <img src="./pictures/puppy.jpg" alt=""></div>
+  <div class="cover back">
+  <img src="./pictures/doggo.jpg" alt=""></div>
```

Comment:
The task is to "display a single card on the page using DOM manipulation." - which is not what is done here. 
Refactor and implement the actual task as described 🙏🏽

Reply:
I changed the code, please see if its okay. I have some issues with my vsc so i sent u a link to playground too